### PR TITLE
Improve README.md

### PR DESCRIPTION
**Pull request by beorn7**
This addresses the confusion in #124.

@brian-brazil @nordbranch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,9 +14,10 @@ Prometheus.
 ## Non-goals
 
 The Pushgateway is explicitly not an _aggregator or distributed counter_ but
-rather a metrics cache. It does not have a statsd-like semantics. The metrics
-pushed are exactly the same as you would present for scraping in a permanently
-running program.
+rather a metrics cache. It does not have a
```
Comment (brian-brazil) [inline README.md]: "It does not have statsd-like..." would read better

File: README.md [hunk 2]
```diff
@@ -250,12 +251,19 @@ processed. Neither is there a guarantee that the pushed metrics are
 persisted to disk. (A server crash may cause data loss. Or the push
 gateway is configured to not persist to disk at all.)
 
+A `PUT` request with an empty body effectively deletes all metrics with the
```
Comment (brian-brazil) [inline README.md]: Hmm, shouldn't that set push_time_seconds still?

File: README.md (hunk 1 above)
Comment (beorn7) [inline README.md]: FIxed.

File: README.md (hunk 2 above)
Comment (beorn7) [inline README.md]: Indeed. So it's not strictly equivalent... Joy. I updated the wording here.

**Comment by beorn7** [comment]
Thanks for reviewing.

**Pull request merged**